daemon/algod/api/algod.oas2.json

```
@@ -1300,6 +1300,66 @@
         }
       ]
     },
+    "/v2/deltas/{round}": {
+      "get": {
```

The private tag is used to determine if the regular or admin token is needed for the endpoint. So it needs to be on all of these
```suggestion
      "get": {
        "description": "Get round deltas for a round.",
        "tags": [
          "private"
        ],
```

---

ACK